### ISSUE TITLE
[accuracy] AllenaiLongformerBase - Minifier does not help as it minifies to `empty`

### ISSUE THREAD

**Issue by anijain2305** [pytorch/torchdynamo#1909]
### 🐛 Describe the bug

`python benchmarks/dynamo/huggingface.py --accuracy --device cuda --backend=inductor --training --float32 --only=AllenaiLongformerBase`

**Comment by ezyang** [comment]
See also https://github.com/pytorch/torchdynamo/issues/1903
